vue/mixins/upload.js

```diff
@@ -36,7 +36,7 @@ export const uploadMixin = {
     },
     computed: {
         noFileSelected() {
-            return this.filesData?.length === 0;
+            return this.filesData?.length === 0 || !this.filesData;

```

Comment:
small simplification :smile: 

```suggestion
            return !this.filesData?.length;

```